AutomatedTesting/Gem/PythonTests/Physics/tests/force_region/ForceRegion_LinearDampingForceOnRigidBodies.py

```diff
@@ -170,6 +170,19 @@ def __init__(self, name):
         and sphere.initial_velocity.IsClose(INITIAL_VELOCITY, CLOSE_ENOUGH)
     )
 
+    # Report the state of the level and then the result of that state
+    Report.info(f"Level Correct Setup: Sphere Initial Y Pos {sphere.initial_pos.y} - Force Region Inital Y Pos "
```

Comment:
I'd suggest reporting the values individually, e.g. report sphere.initial_pos.y and force_region.initial_pos.y separately. Otherwise it's going to be hard to figure out if the problem is that the sphere is incorrectly placed or the force region.

"Level Correct Setup" seems a bit confusing to me, when the values could be incorrect. What about just something like "Level Setup Info"?

As a nitpick, there are a few typos e.g. inital instead of initial in several places.

Reply:
It would be easy to update parts of the calculation above and not update this logging, or vice versa. Perhaps they should be evaluated in the same block?

This could also point to a feature request for EditorTest functionality more similar to GoogleTest's EXPECT_TRUE, EXPECT_EQ, EXPECT_GT, EXPECT_FLOAT_EQ, EXPECT_NEAR, etc.
https://google.github.io/googletest/reference/assertions.html#floating-point

Reply:
I'd also recommend doing individual checks like this to know exactly what failed.

Before:
````
level_correct = (
        (abs(sphere.initial_pos.y - force_region.initial_pos.y) < CLOSE_ENOUGH)
        and (abs(sphere.initial_pos.y - trigger.initial_pos.y) < CLOSE_ENOUGH)
        and (abs(sphere.initial_pos.x - force_region.initial_pos.x) < CLOSE_ENOUGH)
        and (abs(sphere.initial_pos.x - trigger.initial_pos.x) < CLOSE_ENOUGH)
        and (sphere.initial_pos.z > force_region.initial_pos.z > trigger.initial_pos.z)
        and sphere.initial_velocity.IsClose(INITIAL_VELOCITY, CLOSE_ENOUGH)
    )

Report.critical_result(Tests.level_setup, level_correct)
````

After:
````
check1 = abs(sphere.initial_pos.y - force_region.initial_pos.y) < CLOSE_ENOUGH
if not check1:
    Report.info("bla bla")

check2 = abs(sphere.initial_pos.y - trigger.initial_pos.y) < CLOSE_ENOUGH
if not check2:
    Report.info("bla bla")

...

level_correct = check1 and check2 and...

Report.critical_result(Tests.level_setup, level_correct)
````

This way when any of the checks fail it will report the error and only print those that failed.

Reply:
@moraaar @greerdv 

Updated the PR per your feedback.

I went with separating out the conditional checks and conditional validation messaging out since it reads better than keeping them together.

Biggest request here would be whether or not the variable names I used are meaningful enough or if there's some names that would suit these better.

For comparison:
![ConditionalChecksGroupedWithValidationMessaging](https://user-images.githubusercontent.com/29718640/162047529-5c6ba25b-cd71-43e8-b67c-eab59553e6ae.png)

![Uploading ConditionalChecksUngroupedWithValidatiionMessaging.png…]()